metrics/cgroups/oom.go

```diff
@@ -120,6 +122,7 @@ func (o *oomCollector) process(fd uintptr, event uint32) {
 	// being closed make sure we close our copy and remove the container from the set
 	if info.c.State() == cgroups.Deleted {
 		o.mu.Lock()
+		unix.EpollCtl(o.fd, unix.EPOLL_CTL_DEL, int(fd), nil)
```

Comment:
Did you intend to commit this? It looks unrelated.

Reply:
That was my first attempt at fixing the panic.

I don't think it hurts though. I can put it in a different commit.

Reply:
epoll automatically removes it when the fd is closed.  I don't think this is needed and could possibly remove a new fd that is reused.

Reply:
Ok, will remove then.

Reply:
Done.